.buildkite/nightly-benchmarks/nightly-pipeline.yaml

```diff
@@ -71,10 +90,25 @@ steps:
           podSpec:
             <<: *common_pod_spec
             containers:
-              - image: vllm/vllm-openai:latest 
+              - image: nvcr.io/nvidia/tritonserver:24.07-trtllm-python-py3
```

Comment:
Maybe use 24.08?

Reply:
I got r24.07 protobuf template filling scripts from NVIDIA and these scripts doesn't work for r24.08 right now. I confirmed with NVIDIA that in the future there will be a test docker that can be used for benchmarking so I am planning to use r24.07 for now and then switch to the test docker after its release.